packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx

```diff
@@ -1,5 +1,9 @@
+import { config } from '@hicommonwealth/core';
+import { models } from '@hicommonwealth/model';
 import { Button } from 'frames.js/express';
 import moment from 'moment';
+import { mustExist } from 'node_modules/@hicommonwealth/model/src/middleware/guards';
+import { getContestBalance } from 'node_modules/@hicommonwealth/model/src/services/commonProtocol/contestHelper';
```

Comment:
I am not sure if those imports are correct? node_modules? look above on `config` and `models`